crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -102,84 +102,60 @@ impl Format<PyFormatContext<'_>> for FormatLeadingAlternateBranchComments<'_> {
     }
 }
 
-/// Formats the trailing comments of `node`
-pub(crate) fn trailing_node_comments<T>(node: &T) -> FormatTrailingComments
-where
-    T: AstNode,
-{
-    FormatTrailingComments::Node(node.as_any_node_ref())
-}
-
 /// Formats the passed comments as trailing comments
 pub(crate) fn trailing_comments(comments: &[SourceComment]) -> FormatTrailingComments {
-    FormatTrailingComments::Comments(comments)
+    FormatTrailingComments(comments)
 }
 
-pub(crate) enum FormatTrailingComments<'a> {
-    Node(AnyNodeRef<'a>),
```

Comment:
I removed the `Node` variance because this PR removes the last `trailing_node_comments` reference